api/v1/verticarestorepointsquery_types.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright [2021-2023] Open Text.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+// VerticaRestorePointsQuerySpec defines the desired state of VerticaRestorePointsQuery
+type VerticaRestorePointsQuerySpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	// +kubebuilder:validation:Required
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors="urn:alm:descriptor:com.tectonic.ui:text"
+	// The name of the VerticaDB CR that this VerticaRestorePointsQuery is defined for.  The
+	// VerticaDB object must exist in the same namespace as this object.
+	VerticaDBName string `json:"verticaDBName"`
+
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// Optional parameter that will limit the query to only restore points
```

Comment:
Lets use the same xDescriptors marker as you used for VerticaDBName.
```suggestion
        // +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors="urn:alm:descriptor:com.tectonic.ui:text"
	// Optional parameter that will limit the query to only restore points
```